### ISSUE TITLE
Uninitialized constant in Fiddle::Pointer.malloc

### ISSUE THREAD

**Issue by gdelugre** [jruby/jruby#3462]
When allocating memory with Fiddle on JRuby 9.0.3.0, a `NameError` exception is raised because the LibC module is not defined.

```
jruby-9.0.3.0 :001 > require 'fiddle'
 => true 
jruby-9.0.3.0 :002 > Fiddle::Pointer.malloc 16
NameError: uninitialized constant Fiddle::Pointer::LibC
    from org/jruby/RubyModule.java:3181:in `const_missing'
    from /home/user/.rvm/rubies/jruby-9.0.3.0/lib/ruby/stdlib/fiddle/jruby.rb:168:in `malloc'
    from (irb):2:in `<eval>'
    from org/jruby/RubyKernel.java:978:in `eval'
    from org/jruby/RubyKernel.java:1291:in `loop'
    from org/jruby/RubyKernel.java:1098:in `catch'
    from org/jruby/RubyKernel.java:1098:in `catch'
    from /home/user/.rvm/rubies/jruby-9.0.3.0/bin/irb:13:in `<top>'
```

The related code in `stdlib/fiddle/jruby.rb`:

``` ruby
    def self.malloc(size, free = nil)
      self.new(LibC.malloc(size), size, free ? free : LibC::FREE)
    end
```

**Comment by chrisseaton** [comment]
There are other parts of Fiddle missing, for example `Fiddle::Handle::DEFAULT` and `Fiddle::CParser::TYPE_SIZE_T` (which `importer` tries to use for `size_t`).

**Comment by enebo** [comment]
Original PR was reverted to reopening

**Comment by headius** [comment]
I fixed this one issue for 9.2 Any additional issues should be file separately.
